diff --git a/build.rs b/build.rs
index 164abca..abccbbb 100644
--- a/build.rs
+++ b/build.rs
@@ -178,12 +178,7 @@ fn main() {
             .compile("simd");
     }
 
-    let git_output = std::process::Command::new("git")
-        .args(["rev-parse", "HEAD"])
-        .output()
-        .unwrap();
-
-    let git_hash = String::from_utf8(git_output.stdout).unwrap();
+    let git_hash = "@GIT_HASH@";
     println!("cargo:rustc-env=GIT_HASH={git_hash}");
 
     let netplay_id = std::env::var("NETPLAY_ID").unwrap_or("gopher64".to_string());
